ci/e2e/linux/run_tests.sh

```diff
@@ -26,7 +26,7 @@ mkdir -p "${E2E_CACHE_DIR}"
 sudo chown -R $(id -u):$(id -g) "${E2E_CACHE_DIR}"
 
 # to speed up `getenvoy extension build|test`, re-use a single cache across all extensions created by e2e tests
-export E2E_BUILTIN_TOOLCHAIN_CONTAINER_OPTIONS="${E2E_BUILTIN_TOOLCHAIN_CONTAINER_OPTIONS} -v ${E2E_CACHE_DIR}:/tmp/cache/getenvoy -e CARGO_HOME=/tmp/cache/getenvoy/extension/rust-builder/cargo"
+export E2E_TOOLCHAIN_CONTAINER_OPTIONS="${E2E_TOOLCHAIN_CONTAINER_OPTIONS} -v ${E2E_CACHE_DIR}:/tmp/cache/getenvoy -e CARGO_HOME=/tmp/cache/getenvoy/extension/rust-builder/cargo"
```

Comment:
it is less confusing when there aren't extra words in variable names

Reply:
+1